tests/ScheduledExecutionTest.php

```diff
@@ -118,9 +125,10 @@ public function testScheduledExecutionInterval()
     }
 }
 
-
-class TestScheduledDataObject extends DataObject implements TestOnly
+class TestScheduledDataObject extends DataObject
```

Comment:
If I add "TestOnly" to these objects, the tests fail as SapphireTest ignores buildings the database for these objects.

https://github.com/silverstripe/silverstripe-framework/blob/master/src/Dev/SapphireTest.php#L1173

And the DB table is required due to the write occuring in the test
https://github.com/brettt89/silverstripe-queuedjobs/blob/compat-4.0/tests/ScheduledExecutionTest.php#L33